tests.js

```diff
@@ -11,8 +11,56 @@ chai.use(chaiChange)
 /*1: Guess The Number
 Write a program that will generate a number between 1 and 100. Your program will ask the user to guess the generated number.When the user types a guess, the program should tell them if they won (the guess matched the computer's number), if their guess was higher than the number the computer generated, OR if the guess was lower than the number the computer generated. Terminate the program if the user types "exit". */
 
```

Comment:
Hi Deonna, thanks so mch for your reviews..I wasn't done with  them yet and I appreciate the direction!  This was really sooo helpful for me to gauge how much work I have to do (A LOT!)..I have gotten onto codewars and it has similar questions etc. so I'll be workin overtime improving these skills. Can't wait to retake this project on in 2-3 months and see my improvement. Thanks so much for adding this project!!